src/components/video/Video.styles.ts

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components'
+import Image from 'gatsby-image'
+import { colors } from '../../theme/colors'
+
+export const VideoContainer = styled.div`
+  position: relative;
+  padding-bottom: 56.25%;
+  margin-bottom: 2em;
+
+  iframe {
+    position: absolute;
+    height: 100%;
+    width: 100%;
+  }
```

Comment:
This style can be applied directly to the `iframe` as its own `styled-component` 🙂